tests/test_bereikbaarheid/test_resource_utils.py

```diff
@@ -0,0 +1,39 @@
+import datetime
+
+import pytest
+
+from bereikbaarheid.resources.utils import convert_str, convert_to_date, convert_to_time
+
+
+class TestUtils:
+
+    @pytest.mark.parametrize("test_input, expected", 
+                             [("16/03/22 0:00", datetime.datetime(2022, 3, 16, 0, 0)), 
+                              ("2022-02-11 00:00:00.000", datetime.datetime(2022, 2, 11, 0, 0)),
+                              ("22-03-17", ValueError),
+                              ("test", ValueError)])
+    def test_convert_to_date(self, test_input, expected):
+        """Raise an exception if value can't be converted to datetime format"""
+
+        assert convert_to_date(test_input) == expected
```

Comment:
Je test hier alleen de happy flow als het goed gaat. Dit is wat ik met mijn eerdere comment bedoelde; je test niet of het fout gaat als er een verkeerde string wordt gegeven. Daar zou ik dus nog een test voor toevoegen.